### DISCUSSION TITLE
Set URLs for docker containers

### DISCUSSION THREAD

**Discussion by MKr4zy** [louislam/dockge#276] [Ask for Help] [answered]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/dockge/security/policy)

### 📝 Describe your problem

I noticed, that theres a option for "Set URL for a docker Container".
What exactly does this do?

### 🐻 Dockge Version

1.3.3

### 💻 Operating System and Arch

Ubuntu 22.04.3

### 🌐 Browser

Opera Developer 107.0.5004.0

### 🐋 Docker Version

Docker 24.0.5

**Comment by louislam** [comment, marked as answer]
Just a remark for yourself, in case you have any public domains for your services

**Reply by MKr4zy** [reply]
Ahh, thanks! :D
